frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -0,0 +1,261 @@
+                var types = ['c', 'r', 'u', 'd'];
+
+                for (var type in types) {
+                    for (var feature in this.features) {
+                        permissionSet[types[type]].allowed[this.features[feature]] = false;
+                    }
+                }
+
+                this.permissionSets.push(permissionSet);
+            },
+            removePermissionSet: function(index) {
+                this.permissionSets.splice(index, 1);
+            },
+            onClose: function() {},
+            onSubmit: function(submitted) {
+                submitted.permission = countlyAuth.combinePermissionObject(submitted.permission._.u, this.permissionSets, submitted.permission);
+                submitted.password = CountlyHelpers.generatePassword(countlyGlobal.security.password_min);
+                countlyUserManagement.createUser(submitted, function() {
+                // TODO: show toast
+                });
+            },
+            featureBeautifier: function(featureName) {
+                var fa = featureName.split('_');
+                var ret = '';
+                for (var i = 0; i < fa.length; i++) {
+                    ret += fa[i].substr(0, 1).toUpperCase() + fa[i].substr(1, fa[i].length - 1) + ' ';
+                }
+                return ret;
+            },
+            handleCommand: function(command, index) {
+                switch (command) {
+                case "remove-set":
+                    this.removePermissionSet(index);
+                    break;
+                }
+            },
+            setPermissionByFeature: function(index, type, feature) {
+                if (!this.permissionSets[index][type].allowed[feature] && this.permissionSets[index][type].all) {
+                    this.permissionSets[index][type].all = false;
+                }
+            },
+            setPermissionByType: function(index, type) {
+                for (var feature in this.features) {
+                    this.permissionSets[index][type].allowed[this.features[feature]] = this.permissionSets[index][type].all;
+                }
+            }
+        },
+        created: function() {
+            this.permissionSets = [];
+        }
+    });
+
+    // create vue view
+    var ManageUsersContainer = countlyVue.views.create({
+        template: CV.T("/core/user-management/templates/user-management.html"),
+        components: {
+            'data-table': DataTable,
+            'drawer': Drawer
+        },
+        mixins: [countlyVue.mixins.hasDrawers("user")],
+        data: function() {
+            return {
+                users: [],
+                currentTab: 'users',
+                appId: countlyCommon.ACTIVE_APP_ID,
+                drawerSettings: {
+                    title: 'Create new user',
+                    saveButtonLabel: 'Create User',
+                    hasCancelButton: true,
+                    cancelButtonLabel: 'Cancel'
```

Comment:
These two settings are now default in cly-drawer. Hence you wouldn't have to pass them once again.